pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -40,11 +40,16 @@ def __lt__(self, requirement):  # type: (Requirement) -> int
 
 def fix_requirements(f):  # type: (IO[bytes]) -> int
     requirements = []  # type: List[Requirement]
-    before = tuple(f)
+    before = list(f)  # type: List[bytes]
```

Comment:
just an aside, the type annotation isn't required here -- mypy will properly infer that this is a list of `bytes` objects so generally you can leave it out -- I usually prefer to only type annotate things when mypy requires it (the inference often saves some keyboard-typing and importing)

Reply:
I am always worried from this, in case 'read' will return a single stream if its overloaded somehow: 
```
>>> a=b'12'
>>> list(a)
[49, 50]
```

Reply:
I have seen the above happen when 2to3 conversion was mishandled for bytestreams.

Reply:
yeah, fortunately here the `__iter__` function from `IO[bytes]` will give bytes (and mypy knows this)